edge/pkg/edged/edged_status.go

```diff
@@ -423,12 +423,9 @@ func (e *edged) syncNodeStatus() {
 			klog.Errorf("Register node failed: %v", err)
 			return
 		}
-		if err := e.updateNodeStatus(); err != nil {
-			klog.Errorf("Unable to update node status: %v", err)
-		}
-	} else {
-		if err := e.updateNodeStatus(); err != nil {
-			klog.Errorf("Unable to update node status: %v", err)
-		}
+	}
+
+	if err := e.updateNodeStatus(); err != nil {
+		klog.Errorf("Unable to update node status: %v", err)
```

Comment:
@guihecheng Thanks for the fixing! Actually this is not the dup codes, when we first registerNode, then update the nodestatus immediately, or the node will keep not ready for 10s.
And do you have other ways to refactor it?

Reply:
@fisherxu thanks for the explanation for the intent of the code block, but this cleanup will not change the function logic, since the "update block" is simply "merged" and moved outside the else branch. Here I just want to simplify the code a bit :)
Did I miss anything?

Reply:
Ohh, just have a double check you are right :)